app/components/shared/review_component/review_component.html.slim

```diff
@@ -0,0 +1,5 @@
+.review-component
+  .review-component__heading
+    h2.govuk-heading-m.review-component__section id="#{@id}_heading"
```

Comment:
You can drop the 3 `@` here because of `attr_accessor`

Reply:
this will sink in one day!